### PR TITLE
block release promotion if app is updating

### PR DESCRIPTION
**Pull request by awsmsrc**
fixes https://github.com/convox/rack/issues/1185

this behavior was largely already in place

### CODE REVIEW

File: cmd/convox/deploy.go [hunk 1]
```diff
@@ -51,9 +51,9 @@ func cmdDeploy(c *cli.Context) error {
 
 	switch a.Status {
 	case "creating":
-		return stdcli.Error(fmt.Errorf("app %s is still being created", app))
+		return stdcli.Error(fmt.Errorf("deployment blocked, %s is still being created, check `convox apps info`", app))
 	case "updating":
-		return stdcli.Error(fmt.Errorf("app %s is being updated", app))
+		return stdcli.Error(fmt.Errorf("deployment blocked, %s is already updating, check `convox apps info`", app))
```
Comment (MiguelMoll) [inline cmd/convox/deploy.go]: nit: `deployment blocked, foo is still being updated, ...`. Seems consistent with the error above. Thoughts?

Comment (awsmsrc) [inline cmd/convox/deploy.go]: was gonna ask @nzoschke's thoughts on the same thing here, the important thing is pushing towards "check convox apps info"

Comment (nzoschke) [inline cmd/convox/deploy.go]: Yeah we don't need "deployment blocked"

ERROR: app foo is still being created, check `convox apps info`.

File: cmd/convox/releases.go [hunk 2]
```diff
@@ -139,6 +139,15 @@ func cmdReleasePromote(c *cli.Context) error {
 		return stdcli.Error(err)
 	}
 
+	a, err := rackClient(c).GetApp(app)
+	if err != nil {
+		return stdcli.Error(err)
+	}
+
+	if a.Status != "running" {
+		return stdcli.Error(fmt.Errorf("promotion blocked, %s is already updating, check `convox apps info`", app))
```
Comment (nzoschke) [inline cmd/convox/releases.go]: same here

ERROR: foo is already updating, check `convox apps info`

**Pull request closed without merging**